_data/mentors.yml

```diff
@@ -2060,7 +2060,7 @@
   sort: 600
   hours: 4
   type: both
-  index: 61
+  index: 69
```

Comment:
You reverted this change by mistake I think

Reply:
yes, let me fix that